doc/read-the-docs-site/simple example.rst

```diff
@@ -0,0 +1,324 @@
+
+Now we are ready to introduce our main validator function.
+The beginning of the function looks like the following:
+
+.. literalinclude:: AuctionValidator.hs
+   :start-after: BLOCK2
+   :end-before: BLOCK3
+
+Depending on whether this transaction is attempting to submit a new bid or to request payout, the validator validates the corresponding set of conditions.
+
+The ``sufficientBid`` condition verifies that the bid amount is sufficient:
+
+.. literalinclude:: AuctionValidator.hs
+   :start-after: BLOCK3
+   :end-before: BLOCK4
+
+The ``validBidTime`` condition verifies that the bid is submitted before the auction's deadline:
+
+.. literalinclude:: AuctionValidator.hs
+   :start-after: BLOCK4
+   :end-before: BLOCK5
+
+Here, ``to x`` is the time interval ending at ``x``, i.e., ``(-∞, x]``.
+``txInfoValidRange`` is a transaction property.
+It is the time interval in which the transaction is allowed to go through phase-1 validation.
+``contains`` takes two time intervals, and checks that the first interval completely includes the second.
+Since the transaction may be validated at any point in the ``txInfoValidRange`` interval, we need to check that the entire interval lies within ``(-∞, apEndTime params]``.
+
+The reason we need the ``txInfoValidRange`` interval instead of using the exact time the transaction is validated is due to `determinism <https://iohk.io/en/blog/posts/2021/09/06/no-surprises-transaction-validation-on-cardano/>`_.
+Using the exact time would be like calling a ``getCurrentTime`` function and branching based on the current time.
+On the other hand, by using the ``txInfoValidRange`` interval, the same interval is always used by the same transaction.
+
+The ``refundsPreviousHighestBid`` condition checks that the transaction pays the previous highest bid to the previous bidder:
+
+.. literalinclude:: AuctionValidator.hs
+   :start-after: BLOCK5
+   :end-before: BLOCK6
+
+It uses ``PlutusTx.find`` to find the transaction output (a UTXO) that pays to the previous bidder the amount equivalent to the previous highest bid, and verifies that there is at least one such output.
+
+``singleton adaSymbol adaToken amt`` constructs a ``Value`` with ``amt`` Lovelaces (the subunit of the Ada currency).
+``Value`` is a multi-asset type that represents a collection of assets, including Ada.
+An asset is identified by a (symbol, token) pair, where the symbol represents the policy that controls the minting and burning of tokens, and the token represents a particular kind of token manipulated by the policy.
+``(adaSymbol, adaToken)`` is the special identifier for Ada/Lovelace.
+
+The ``correctNewDatum`` condition verifies that the transaction produces a *continuing output* containing the correct datum (the new highest bid):
+
+.. literalinclude:: AuctionValidator.hs
+   :start-after: BLOCK6
+   :end-before: BLOCK7
+
+A "continuing output" is a transaction output that pays to the same script address from which we are currently spending.
```

Comment:
I thought a continuing output is an output with the same address, value, datum and reference script as the original output, but with a different id. Am I wrong?

Reply:
If all those stay constant why have a transaction :)
I think an output paying to the same script is accurate. Value and datum will often change, and reference script should probably be kept somewhere else for the long term.

Reply:
I think you meant this to be a response to a different comment.

I take your point, but I think I disagree. I would say that it _is_ a tutorial and so it should live with the tutorials, or else we should throw away that organizational structure. But I don't like being in the situation where we have the organizational structure but we're using it inconsistently, that makes it very hard to know where to put things. _However_ this is your call I think, I'm just registering my thoughts.

Putting it under tutorials doesn't prevent us from linking to it specially, though.

Reply:
@michaelpj This page is more like "getting started", which is usually separate from "tutorial/user guide".

We don't call it "getting started" because there's no easy way for someone to _actually_ get started - we don't have a browser-based playground, or a simple way to set up a local environment, or things like that. But the bottom line is I want this to be the very first thing one reads when coming to the RTD site.